wowup-electron/src/app/services/electron/electron.service.ts

```diff
@@ -17,6 +18,7 @@ import { ValueResponse } from "../../../common/models/value-response";
 export class ElectronService {
   private readonly _windowMaximizedSrc = new BehaviorSubject(false);
   private readonly _windowMinimizedSrc = new BehaviorSubject(false);
+  private readonly _rendererEventSrc = new BehaviorSubject('');
```

Comment:
This might need a new name. doesnt really describe what it does.

Reply:
Hopefully the new name better describes the use.